src/Aquifer.API/Endpoints/Resources/Content/SendForPublisherReview/Community/Request.cs

```diff
@@ -0,0 +1,6 @@
+namespace Aquifer.API.Endpoints.Resources.Content.SendForPublisherReview.Community;
+
+public class Request
+{
+    public int? ContentId { get; set; }
```

Comment:
This probably can be `int` not optional since once the second path in the endpoint is removed like my above comment, it'll always be part of the path.